### ISSUE TITLE
Override implementation in PanelBuilders.table() for dashed lines

### ISSUE THREAD

**Issue by korinekgergely** [grafana/scenes#623]
Please someone help me, write me the exact implementation of the override attached below.
I need it in a `PanelBuilders.table() `
Because I have searched the source files, I found this everywhere:

`/**
 * TODO docs
 */`

![Monosnap Edit panel - Template Dashboard !!!! DO NOT DELETE !!!! - Dashboards - Grafana 2024-02-27 15-12-48](https://github.com/grafana/scenes/assets/2194896/02cf2d3b-95a0-4360-a0a8-4e3dee4dd1d4)

Or as json:
`"overrides": [
      {
        "matcher": {
          "id": "byName",
          "options": "CRITICAL"
        },
        "properties": [
          {
            "id": "custom.lineStyle",
            "value": {
              "dash": [
                10,
                10
              ],
              "fill": "dash"
            }
          },
          {
            "id": "custom.fillOpacity",
            "value": 0
          },
          {
            "id": "color",
            "value": {
              "fixedColor": "dark-red",
              "mode": "fixed"
            }
          },
          {
            "id": "custom.hideFrom",
            "value": {
              "legend": true,
              "tooltip": false,
              "viz": false
            }
          }
        ]
      },
      {
        "matcher": {
          "id": "byName",
          "options": "WARNING"
        },
        "properties": [
          {
            "id": "custom.lineStyle",
            "value": {
              "dash": [
                10,
                10
              ],
              "fill": "dash"
            }
          },
          {
            "id": "custom.fillOpacity",
            "value": 0
          },
          {
            "id": "color",
            "value": {
              "fixedColor": "yellow",
              "mode": "fixed"
            }
          },
          {
            "id": "custom.hideFrom",
            "value": {
              "legend": true,
              "tooltip": false,
              "viz": false
            }
          }
        ]
      }
    ]`

I have a try but I don't find the implementation of the dashed lines. I am not sure if it is a good start:
`.setOverrides((critical: any) =>
      critical
        .matchFieldsWithName('CRITICAL')
        .overrideColor({
          mode: 'fixed',
          fixedColor: 'red'
        })`
Thank you in advance.

**Comment by torkelo** [comment]
The screenshot is for overrides for the time series visualization, not table.

So confused

**Comment by korinekgergely** [comment]
Of course, sorry, my fault. This is a timeseries()

**Comment by korinekgergely** [comment]
But the question remains, rightly in the case of timeseries.

**Comment by korinekgergely** [comment]
@torkelo 
This is the goal, this is what it should look like.
![Dashboards - Grafana 2024-02-28 07-50-04](https://github.com/grafana/scenes/assets/2194896/40cfc072-daf7-464b-87cf-8057e1c63ce5)

**Comment by torkelo** [comment]
and the 90% red line and yellow lines are time series?

**Comment by korinekgergely** [comment]
@torkelo 
Yes, the same time series as the others. I just want to display certain Field names as dashed. What appears as a dashed line will typically be a straight line for us, but not necessarily.
Sorry for the late reply. The question is just as interesting and important.

So again, to briefly summarize the question: how can I change the style of a time series to dotted by identifying it by field name (as shown in the top image)? 
PanelBuilders.timeseries()
I suspect that an .override... is needed, but I couldn't find a lineStyle in the documentation. Maybe there is a different way.

**Comment by korinekgergely** [comment]
Found the solution:
`.setOverrides((a: any) =>
      a
        .matchFieldsWithName('nameBlaBla')
        .overrideCustomFieldConfig('fillOpacity', 0)
        .overrideColor({
          fixedColor: 'red',
          mode: 'fixed'
        })
        .overrideCustomFieldConfig('lineStyle', {
          "dash": [
            10,
            10
          ],
          "fill": "dash"
        })`
